MeetingManagerFront/meeting-manager-front/src/components/Content/MeetingsList.vue

```diff
@@ -0,0 +1,55 @@
+<script>
+import axios from "axios"
+import MeetingView from './MeetingView.vue';
+
+export default {
+    components:
+    {
+        MeetingView
+    },
+
+    data()
+    {
+        return{
+            meetings: []
+        }
+    },
+
+    mounted() 
+    {
+        this.getAllMeetings();
+    },
+
+    methods:
+    {
+        getAllMeetings()
+        {
+            axios.get("https://localhost:44343/api/meetings/")
```

Comment:
You can create folder api with file index.js and add all api methods.
const api = {
user: {
get: {
      user(userId) {
        return axios.get('https://localhost:44343/api/user', {
          params: { userId }
        })
      },
},
meeting: {
}
}